Tests/Block/Service/AbstractCategoriesBlockServiceTest.php

```diff
@@ -0,0 +1,121 @@
+
+/**
+ * Class AbstractCategoriesBlockServiceTest.
+ *
+ * @author Christian Gripp <mail@core23.de>
+ */
+final class AbstractCategoriesBlockServiceTest extends AbstractBlockServiceTest
+{
+    /**
+     * @var ContextManagerInterface
+     */
+    private $contextManager;
+
+    /**
+     * @var CategoryManagerInterface
+     */
+    private $categoryManager;
+
+    /**
+     * @var CategoryAdmin
+     */
+    private $categoryAdmin;
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->templating = new FakeTemplating();
+        $this->contextManager = $this->getMock('Sonata\ClassificationBundle\Model\ContextManagerInterface');
+        $this->categoryManager = $this->getMock('Sonata\ClassificationBundle\Model\CategoryManagerInterface');
+        $this->categoryAdmin = $this->getMockBuilder('Sonata\ClassificationBundle\Admin\CategoryAdmin')->disableOriginalConstructor()->getMock();
+    }
+
+    public function testDefaultSettings()
+    {
+        $blockService = $this->getMockForAbstractClass('Sonata\ClassificationBundle\Block\Service\AbstractCategoriesBlockService', array(
+            'block.service', $this->templating, $this->contextManager, $this->categoryManager, $this->categoryAdmin,
+        ));
+        $blockContext = $this->getBlockContext($blockService);
+
+        $this->assertSettings(array(
+            'title' => 'Categories',
+            'category' => false,
+            'categoryId' => null,
+            'context' => 'default',
+            'template' => 'SonataClassificationBundle:Block:base_block_categories.html.twig',
+        ), $blockContext);
+    }
+
+    public function testLoad()
+    {
+        $category = $this->getMockBuilder('Sonata\ClassificationBundle\Model\CategoryInterfacer')
+            ->setMethods(array('getId'))
+            ->disableOriginalConstructor()
+            ->getMock();
+        $category->expects($this->any())->method('getId')->will($this->returnValue(23));
+
+        $this->categoryManager->expects($this->any())
+            ->method('find')
+            ->with($this->equalTo('23'))
+            ->will($this->returnValue($category));
+
+        $block = $this->getMock('Sonata\BlockBundle\Model\BlockInterface');
+        $block->expects($this->any())
+            ->method('getSetting')
+            ->with($this->equalTo('categoryId'))
+            ->will($this->returnValue(23));
+        $block->expects($this->once())
+            ->method('setSetting')
+            ->with($this->equalTo('categoryId'), $this->equalTo($category));
+
+        $blockService = $this->getMockForAbstractClass('Sonata\ClassificationBundle\Block\Service\AbstractCategoriesBlockService', array(
+            'block.service', $this->templating, $this->contextManager, $this->categoryManager, $this->categoryAdmin,
+        ));
+        $blockService->load($block);
+    }
+
+    /**
+     * @covers Sonata\BlockBundle\Tests\Block\AbstractCategoriesBlockService::preUpdate
```

Comment:
The Symfony way is to not use `@covers` at all and let the coverage cover the called code.

This was discussed on another PR. I have to update the code on dev-kit about that.